core/src/main/res/values/strings.xml

```diff
@@ -41,6 +41,21 @@
     <string name="core_date_format_MMMM_dd">MMMM dd</string>
     <string name="core_full_date_with_time">dd MMM yyyy hh:mm aaa</string>
 
+    <!-- region: Course Date types -->
+    <!-- Label used when the date is Today -->
+    <string name="date_type_today" translatable="false">Today</string>
+    <!-- Label used when an assignment is for verified user only -->
+    <string name="date_type_verified_only" translatable="false">Verified Only</string>
+    <!-- Label used when an assignment is completed -->
+    <string name="date_type_completed" translatable="false">Completed</string>
+    <!-- Label used when an assignment is past due -->
+    <string name="date_type_past_due" translatable="false">Past Due</string>
+    <!-- Label used when an assignment is due next -->
+    <string name="date_type_due_next" translatable="false">Due Next</string>
+    <!-- Label used when an assignment is not yet released -->
+    <string name="date_type_not_yet_released" translatable="false">Not Yet Released</string>
```

Comment:
We follow the "modulename_variablename" naming convention to avoid problems with name conflicts in other modules.
`name="date_type_today"` -> `name="core_date_type_today"`
`name="date_type_verified_only"` -> `name="core_ date_type_verified_only"`
etc.